MAUI/Autocomplete/AutoSizing.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: AutoSizing in .NET MAUI Autocomplete control | Syncfusion
+description: Learn all about autosizing support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more here.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# AutoSizing in .NET MAUI AutoComplete (SfAutocomplete)
+
+AutoSizing can be enabled in [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control so that the control will extend its layout based on the input content.
```

Comment:
AutoSizing can be enabled in the [SfAutocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control so that the control will extend its layout based on the input content.